src/Libraries/RevitNodes/Elements/SunSettings.cs

```diff
@@ -105,8 +105,27 @@ public DateTime EndDateTime
         /// </summary>
         public DateTime CurrentDateTime
         {
-            get { return InternalSunAndShadowSettings.ActiveFrameTime; }
+            get { return TranslateTime(InternalSunAndShadowSettings.ActiveFrameTime); }
         }
 
+        /// <summary>
+        /// Fix for: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8993
+        /// 
+        /// StartDateAndTime, EndDateAndTime and ActiveFrameTime should return local datetime,
+        /// that is set in Revit project configuration.
+        /// But it returns customized datetime + user timezone offset, which is incorrect.
+        /// Althought in documentation it's said, 
+        /// that "The output value will be in Coordinated Universal Time (UTC), 
+        /// but input may be in local time as well."
+        /// There is no means to return local time.
+        /// </summary>
+        internal static DateTime TranslateTime(DateTime utc)
+        {
+            // Get user local hours offset.
+            var offset = TimeZoneInfo.Local.GetUtcOffset(DateTime.UtcNow).Hours;
```

Comment:
this makes sense to me, what happens with daylight savings time? I don't think UTC has it, does the TimeZoneInfo take care of it?